documentation/configuration.md

```diff
@@ -327,69 +434,119 @@ Match pid 1 (Shorthand)
 Due to limitations in event counters, `dotnet monitor` supports only **one** refresh interval when collecting metrics. This interval is used for
 Prometheus metrics, livemetrics, triggers, traces, and trigger actions that collect traces. The default interval is 5 seconds, but can be changed in configuration.
 
-```json
-{
-    "GlobalCounter": {
-      "IntervalSeconds": 10
-    }
-}
-```
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+      "GlobalCounter": {
+        "IntervalSeconds": 10
+      }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_GlobalCounter__IntervalSeconds="10"
+  ```
+</details>
 
 ### Metrics Urls
 
 In addition to the ordinary diagnostics urls that `dotnet monitor` binds to, it also binds to metric urls that only expose the `/metrics` endpoint. Unlike the other endpoints, the metrics urls do not require authentication. Unless you enable collection of custom providers that may contain sensitive business logic, it is generally considered safe to expose metrics endpoints. 
 
-Metrics urls can be configured via the command line:
+<details>
+  <summary>Command Line</summary>
 
-```cmd
-dotnet monitor collect --metricUrls http://*:52325/
-```
+  ```cmd
+  dotnet monitor collect --metricUrls http://*:52325/
```

Comment:
Let's remove the trailing `/` so its consistent with the json and env var examples.